sygnal/apnstruncate.py

```diff
@@ -26,7 +26,7 @@ class BodyTooLongException(Exception):
     pass
 
 
-def is_too_long(payload, max_length=2048):
+def is_too_long(payload: Dict[Any, Any], max_length: int = 2048) -> bool:
```

Comment:
Looks like the type annotation you want is `dict` here.

Reply:
Now that you've changed that with https://github.com/matrix-org/sygnal/pull/225/commits/8ae8f5212c77ab3899cf5577e8ebfb5f45b8493c you'll also want to remove the import to `typing.Dict` on line 18 since it's not used anymore.

Reply:
Thank you, I have just removed it after seeing my test fails.

Reply:
@babolivier is there a difference between `typing.Dict` and `dict` on python versions? 
while running `mypy --strict sygnal/ tests/ setup.py` I'm getting an error :point_down: 
![dict](https://user-images.githubusercontent.com/50038533/119662797-74175980-be3a-11eb-9435-d389e57c0a92.png)
I'm running `python 3.8`
https://github.com/matrix-org/sygnal/blob/656c0ead8750a5630080a6bd0c63b1bae52a478f/sygnal/apnstruncate.py#L29